.github/hacktoberfest.md

```diff
@@ -21,4 +21,5 @@ To have your review of a translation PR recognized for Hacktoberfest, open a PR
 These awesome people reviewed i18n PRs for us. Thank you! 💜
 
 - @agustinmulet reviewed #1609
-- 
+- @moustaphadev reviewed #1654
```

Comment:
```suggestion
- @moustaphadev reviewed #1668
```

Reply:
I think this was #1668 ?  😄 I'm going to give you credit for that one, anyway!

Reply:
I reviewed both, although #1668 hasn't been merged yet